main.go

```diff
@@ -268,6 +268,27 @@ func run() int {
 	app.Usage = "Memo Life For You"
 	app.Version = VERSION
 	app.Commands = commands
+	app.Action = func(c *cli.Context) error {
+		args := c.Args()
+		if args.Present() {
+			xcmd := fmt.Sprintf("memo-%s", args.First())
+			xcmdpath, err := exec.LookPath(xcmd)
```

Comment:
Is `$memodir/plugins` the `~/.config/memo/_posts/plugins` at default?
or `~/.config/memo/plugins`?

In the config file, `memodir` is the directory stores memo text files that users create. And the default is `~/.config/memo/_posts`. Do you intend to place plugins directory in the same directory that contains memo text files?